docs/root/version_history/current.rst

```diff
@@ -87,6 +87,26 @@ Removed Config or Runtime
 
 New Features
 ------------
+* config: added new runtime feature `envoy.features.enable_all_deprecated_features` that allows the use of all deprecated features.
```

Comment:
Why so many features are included?

Reply:
not sure, cherry-pick probably updated this file..

Reply:
Is this a clean cherry-pick? The original pr is not adding so many text about features.

Reply:
Yeah there was not even a single conflict

Reply:
we can remove the other lines if you want...

Reply:
The code is exactly the same so I assume it is ok.